client/state/utils.js

```diff
@@ -19,3 +23,34 @@ export function isValidStateWithSchema( state, schema, checkForCycles = false, b
 	}
 	return result.valid;
 }
+
+export function createReducer( actionReducers, defaultState = null, schema = null ) {
+	const defaultActionReducers = {
+		[SERIALIZE]: ( state ) => {
```

Comment:
I'm aware _SERIALIZE_ AND _DESERIALIZE_ action reducer could have shorter form, but I wanted make them easier to read. We can always refactor :)